samples/matter/template/sample.yaml

```diff
@@ -73,6 +73,7 @@ tests:
     build_only: true
     extra_args: SB_CONFIG_WIFI_NRF700X=y template_SHIELD=nrf700x_nrf54h20dk
       CONFIG_CHIP_WIFI=y CONFIG_CHIP_BOOTLOADER_NONE=y
+      CONFIG_CHIP_OTA_REQUESTOR=n SB_CONFIG_MATTER_OTA=n
```

Comment:
Actually, it should work because it was repaired here: https://github.com/nrfconnect/sdk-nrf/pull/16398/commits/bea1debf0b01e837335f969a7b63c8f382cbfb7d
Could you verify it and possibly remove those configs?

Reply:
Discussed f2f, I tried to enable DFU for this build, but it didn't fit flash.